zeeguu_api/api/teacher_dashboard.py

```diff
@@ -432,3 +436,81 @@ def update_cohort(cohort_id):
     except sqlalchemy.exc.IntegrityError:
         flask.abort(400)
         return 'IntegrityError'
+
+
+@api.route("/upload_files/<cohort_id>", methods=["POST"])
+@with_session
+def upload_file(cohort_id):
+    '''
+        uploads a file for a cohort with input from a POST request
+    '''
+    if (not has_permission_for_cohort(cohort_id)):
+        flask.abort(401)
+    try:
+        for file in json.loads(request.data):
+            url = Url('teacher-dash-file/{}'.format(datetime.now()))
```

Comment:
I propose `Url('user-article-{}'.format(...)`. what do you think?

Reply:
also, that will have spaces in the name... 
e.g. 

    'user-article-2019-04-02 14:38:07.416790'

is this what you indended?

Reply:
maybe something like this?

    'userarticle/{}'.format(uuid.uuid4().hex)